src/shared/constant/index.ts

```diff
@@ -3,6 +3,8 @@
R9O7+JIuYXVJe8PtQvr7rXnRfHGIzWNj9Y+j9L59sVZNIYieqPndvVALDuHATrKnfyGfl85gnWw3qd3x80gRgfyEWgYQi6/IuqTmj5kaGKw7nWcKqzCSPOGL6M1cbuDx1rJ1x1j1XIwy/abP8IQUxaihbROF+MvPvP7VL6+q/D0iJr/s58ry8ZHaj41FFg9To5kjX7B8j1HWF1k93PubT60UrNZYd5Ar8Zmz87JQUjWcUupLUbMLy0k0+h0mQJIjoFhlUxBP1ELhmEvn4AZAn7xcbRpjAFNKV5Hu5lUMpopKwaowPZlhxQbh+6/B77hOq/ABDMYVGoXA5SqritOTFePy4lK1/p//12mwlHPz+V7wGR+ozxvn5nKGUHcILGkRHr9ZUBqLkA1c0vwlg7OryS36S0J4k7LrnYHhtr1Z+3cdwAQBXqgkbdgOCPAEE9TZeU0mwK86BVhoxFiBuYzahAxBWDNTqAwdpH4EwWd3WfAwQ//N6PRhMh3pelOe5N6+z/Q/4Pln9qt+vdcuWuYB8cNsPB4HGrV1j65D3YDq5kPMpuXmB/xTLtxxjnz+RHKGuEtaTNKhe+GB7r4dZ/ZL7HhXCNQyILdBn8LpS9tu0jdmtuRIbSx/YvZBlfrmsABMfDtRuLhWQG/JUmKCIt+mLHxRjff5g+MBwgCCA54RUmH56f5adwAyhBI9S1pwB9nSGt34RxX+y/0+4oNzF5S6osOpmfIJ5sp8HI9yD1x6mjRis0zgyUlJOWOQhUj3jl8YBBGqHGT9gWWBRgxGdntV1mN+wqyc/bN+4AoIp15SCY3kK/XbkVwu6Jtq/A4JIIi0ZguZi5ZBHZGc0qKmHu4/zAgJaA+frRaDFEWqChmgk7qPz3AXgnDPof8v2Y+AfZJvYYR8Hp18OmYdXv05rYMaS/ivOAZkuq6c9rkaFpznR8B6B8mN7QSI10yhPzeZUOtDiz7fJdz3zqsnOWsAoYLdcVQXAMIMgAAczY05nGZ9eBkdpJFI6e8X0EMJn4hiZQOL+HsdIcDhj8wJwABrYJMwNH3w1zE7abOUWw7urYC+rXC5OZm5cUy9ikSHUhOhm0gNrPpVzdjH7100EoIPINgOwAP0FUz/L+Gr+VpbF9FRvhzA4xv0x0eXTW+LeHJ1R3fLpuJKUL2t29uv/+pFg6GQY8DQd8iEiN0QHRyubGJwV0T1iqWQGF5Tyf8iscr6i+VYasc5+q1fBtI18k6WeuyahP1wsy6TkxpMZZAVVOvRAGPF4F6Qbf5x8U5tKaTgJ4fHhKlI9deIdmOAuaomY04sxfOk6YTwu8m5Q/Ll13DoKn++/PCqWTYcjTVFQ/6eq7YwIPAqW1iuD3kL0ZBDlQ8rR4L89XHvBzJgoI2o+TMGmjXwLz4Sdx2r+gYWmY2OHGASQ/Mt3LVHkqjET5DBUECM1BFOnwdRbVmTP+2W4Q8+eOA8mnuu7GrQbIK9jdBIJCgiYwe1paHKmP08YQnMxOfDEg1w55HMBwhuX3ebo/FPJLgv2WP2IqWVWea4icqcojpjbeo7wDZYb360E9ShpA8A9M+Gn451H98/aC+Zo1HE/MV2XHPQBUye4IgtvRBPTIaILkaXX/UDpOGzvDAhNDUAzJeRXCYgoMVHn4DYYrLOcgaFLpis+BkTNaD+VhdQ/5C+QrL4muNchTxXbQgVO/QvqPn/kv1j1i3xiosiPoJgQA1N7uJhAUHQT2OAOBOG2MTlCPKyZIct2LEgw3SHPJJbrBNBgmvvkjzlSp/SYJj4BogEClawbIn4kPbplHryIuYWRRXF+xWzAITzzoC/aQG3xxAkl1GE/OJWI8VWiounTRH3cDhhNn2ouR528xHNQRKjpiJvUxODc+PATa+UigMRogjmGdzyXAnDCZFPL5UFHP59a9l4EFr2Gdhn5u3RMmPUxEEW5Ow9qvklbsYNRA33TlEV+291EVG6+Sr7rJ0YyJ5bqhrzZ4/PBJe2S36daFKv4uzGJnkXf2QTMjoS7hktTocjXtYi9JJ55Lt7r8XhFatXPJJj2X8MazxOV2gIp1zRE0T5WfuyqynLuB3xR4W4P5ZfA2TiVf9ZebcABQpRsbPJZZ/62PZ2diCHwE9msDhVbhmOWNTHVMREYRhr9+5eDw+xgJk8PwLdy7Iad3ick8JAy4r35ecQIH8UxTW6JZRbZ+LSX+mGO+at9E5WvEkDDO98f16Hh1kGVCOwHYifzafe21aIIv8mn3i5FSqfQ6aruQq/+8O5AqJ5xPKeczh5qi83UD1HpjEUfdgCZyfDKHsD8bugXNEiYcK5syP7CO8i6btzicI7C0i2Fe966f4BkuV0XAiezEfATN0pses5uQxCOYcPkKkqlZP2kDCWpNEq+AxFiXh3mQYB7nEs69xu6OKM/bnB60gPb41Zkm1tdNKVb+NZzjc2zOfC0vTyTm07wwLFJgorsuBLQb+0DtePMB9hpU84eZzp0nqZVDSqsIr/bnpa4BiMffzBDUfS7tmtqVUci9pF3TwkWtItLNiGo/4nPyK878rt2gsiX1Xd3xww9FTCA30buALUmdlonh8m7h7APtzUzDvpfuoEvz/IF33j3IktcagBiqefwwYsDXugHMDnFoERZwTM9i2fcCnh9j6X/2bT+w6/ViaQxeNiH6etV3a+6FBgBvo0YJGIoCgbS88e3gCUjzm9AKr2Vq9hAf6tFy79Mj8xWG+Y3hnsBBATop7C6euZVnv/OupfZzfwF/pO7zzSF53ET0X5AAiIzQ595pMxDOeYlNYWL/UFbrjkYzHInKnw3jO+ka2pHufpjej7CvIX05/L+X06dv37TBHikz2lCZZco7bAGfkesT8giu+K4/eOOZAtIIugaro3YMl2HuYOnlLiuWy4OnD/bcRIh/IWuArdFfI4ZkmfpuEDHnWUYQGG9k9K6CNJdwPYjmsDmLLbuEtBeytP9vA4B4u6VrpoEDYcsMf7j/AwVesBT4/0slhdQjEBhrAAAAAElFTkSuQmCC';
```

Comment:
This one was converted from https://github.com/unisat-wallet/extension/blob/master/build/_raw/images/logo/logo%40128x.png